ledger-core/v2/reference/local.go

```diff
@@ -120,6 +120,11 @@ func (v Local) asEncoderReader(limit uint8) *byteReader {
 	return &byteReader{v: v, s: limit}
 }
 
+//DEPRICATED
```

Comment:
should be
// deprecated
don't forget the space

Reply:
Fixed